### ISSUE TITLE
[ Design/Discovery ]Bad Address Indicator Design Work

### ISSUE THREAD

**Issue by Justin-Pickett** [department-of-veterans-affairs/va.gov-team#2198]
## Background

This ticket will house the intial design/discovery work for Bad Address Indicator. 

## Goal 

Upload the design/discovery UX work for bad address indicator work

## Definition of Done

- [x] Design discovery
- [x] UX design work has been completed and uploaded to this ticket. 
- [x] UX design work has been shared with the team. Feedback gathered
- [x] UX design work has been shared with copy/content team for review and feedback. 
- [ ] Feedback changes have been completed if needed.
- [ ] Proceed to create ticket and schedule user testing of Bad Address Indicator work

**Comment by Justin-Pickett** [comment]
This is currently a placeholder until we begin design discovery for Bad Address Indicator. Will fill this out accordingly in the coming days.

**Comment by artistgreen** [comment]
-[] Design Research

**Comment by Justin-Pickett** [comment]
@lkoenigsberg will follow up with some more data on this work. She will advise us if we will continue with any UX work regarding Bad Addresses. Stay tuned.

**Comment by artistgreen** [comment]
@lkoenigsberg, @Justin-Pickett  - a few questions popped up in design discovery and was hoping you can answer them or point me in the right direction.

- _Does the VA have a process of conveying urgent messaging to Veterans via email, phone or both?_  
I have received feedback that we send out email notifications for 2FA and connecting accounts.  Conceptually, if we did send email we could notify Veterans about their bad address and place a link to the Change Address page.

_I was advised that most online forms pre populate with Veteran info from their profile.  Is this accurate?_
If it is accurate, we should consider displaying an alert message in each form.

There is discussion around the concept of setting a 'threshold' for returned mail items.  If we can set one, say it is 5 pieces of returned mail, then we can display warning alerts for the first 5 pieces and if the number of returned items goes beyond the threshold number, we escalate the messaging to more of an 'error' message.  
Is this even worth considering?  I am asking because this concept was well received in discussion.

Can discuss anytime - thank you @lkoenigsberg and @Justin-Pickett

**Comment by lkoenigsberg** [comment]
Thanks for the questions, here’s some insights:

There are a few emails that VA.gov sends our, as you point out and now also Direct Deposit. That’s a way of saying we have a way of sending emails.

Yes, if your logged in the forms do pre-populate with the address on file. The user has the ability to change that pre-filled content with a new address but that information is not then used to update the master record/profile today. Be an interesting exploration.

We can talk to the VEO about what the current threshold is, if any. We don’t directly control this and my understanding is that it’s just an on/off and not associated with any threahold.

Happy to discuss further if needed

**Comment by artistgreen** [comment]
Thanks @lkoenigsberg!

Great to hear the VA uses email for some messaging.  I think there is a good case to consider email for this type of messaging.  Veterans may not be receiving disability or pension payments on time and if I was a Veteran in this situation, I'd like to know ASAP.  The email message could have a link to the 'Change Address' page where users can be instructed on how to change their address.

Hmmm, interesting.  So the Veteran can update their info, in this case mailing address, within the form but that information is not updated across systems or to the master record.  Maybe this use case could lead to an effort to make that happen.  If I was a Veteran, I imagine I would think if I updated my info in the form, the VA would have that as updated info for me everywhere in their system. (Unless there is messaging to advise me otherwise - which there could be, I haven't looked in any forms yet).  Will plan to explore further if you think it's worth the time.

OK - the threshold concept was more creative thinking about how we could use the escalation in messaging as an impactful method of getting the Veteran's attention.  Probably a feature enhancement down the road? :-)  Can continue to discuss with the proper people if we think this could be a valuable consideration.

Would love to catch up and discuss at your convenience - let me know if you are open at all tomorrow.  Can show you the concept workflows and a few screens also.

Thank you Lisa!

**Comment by artistgreen** [comment]
Current workflow options based on research:

https://invis.io/KQUMDRQXN9H#/391507241_Home

There are currently 4 options:

1.  Email notification    >    Direct user to Change Address page    >    User Signs in    >    Alert Displays on VA.gov Homepage   >   User is directed to Profile page

Pros:  
	• 	VA can initiate communication to the veteran
	• 	Directing user to the Change Address page presents them with guidance on how to update their address
	• 	Alert on VA.homepage is clear and evident
	• 	Alert messaging may be able to be personalized with address on file and/or what the Veteran is receiving in the mail
	• 	Alert CTA allows easy linking to Profile page to ensure users know where to go

Cons:  
	• 	We may not be able to use email as a form of communication
	• 	VA.homepage hijack to display alert may be confusing since it is something the user does not normally see

--------------------

2.  User arrives at VA.gov and Signs in    >    Alert Displays on VA.gov Homepage   >   User is directed to Profile page

Pros:  
	• 	Alert on VA.homepage is clear and evident
	• 	Alert messaging may be able to be personalized with address on file and/or what the Veteran is receiving in the mail
	• 	Alert CTA allows easy linking to Profile page to ensure users know where to go

Cons:  
	• 	We have to wait for the user to arrive at and sign in to VA.gov
	• 	VA.homepage hijack to display alert may be confusing since it is something the user does not normally see

--------------------

3.  User arrives at VA.gov and Signs in    >    Modal Displays on VA.gov Homepage   >   User is directed to Profile page

Pros:  
	• 	Modal on VA.homepage is clear and evident
	• 	Modal messaging may be able to be personalized with address on file and/or what the Veteran is receiving in the mail
	• 	Modal CTA allows easy linking to Profile page to ensure users know where to go

Cons:  
	•	We have to wait for the user to arrive at and sign in to VA.gov
	•	VA.homepage hijack to display modal may be confusing since it is something the user does not normally see
	•	May be too ‘in your face’ to users with the modal/overlay

---------------------

4.  User arrives at VA.gov and Signs in    >    No messaging on VA.gov Homepage   >   User opens User menu (maybe we show an indicator by the Profile link)  >   Profile page where Inline Alert is displayed

Pros:  
	• 	Modal on VA.homepage is clear and evident
	• 	Modal messaging may be able to be personalized with address on file and/or what the Veteran is receiving in the mail
	• 	Modal CTA allows easy linking to Profile page to ensure users know where to go

Cons:  
	•	We have to wait for the user to arrive at and sign in to VA.gov
	•	VA.homepage hijack to display modal may be confusing since it is something the user does not normally see
	•	May be too ‘in your face’ to users with the modal/overlay

**Comment by artistgreen** [comment]
These areas are being considered to display messaging:

1.  VA.gov homepage
2.  https://www.va.gov/change-address/
3.  https://www.va.gov/records/download-va-letters/  and/or   https://va.gov/records/download-va-letters/letters/confirm-address
4.  https://va.gov/health-care/how-to-apply/
5.  https://va.gov/pension/how-to-apply/
6.  https://va.gov/disability/how-to-file-claim/
7.  https://va.gov/housing-assistance/home-loans/how-to-apply/
8.  https://va.gov/health-care/get-medical-records/
9.  https://va.gov/health-care/refill-track-prescriptions/. (Important note: If you change your address within My HealtheVet, it does not change your address for prescription shipments. Please contact the VA health facility where you get care to have them update your address on file.)
10.  https://va.gov/my-va/
11.  https://va.gov/health-care/my-health-account-validation/error/needs-va-patient
12.  https://va.gov/account/
13.  https://va.gov/profile/

**Comment by artistgreen** [comment]
Another consideration is to display messaging with an inline alert inside forms. 

Currently form info gets pre-populated from the profile - but that information is not then used to update the master record/profile today

Example screen from Contact Information form:

![image.png](https://images.zenhubusercontent.com/5d49d1baa8585c0001b6a7a8/40c9a7e2-ad39-4c8b-8716-4c85db95e10f)

**Comment by Justin-Pickett** [comment]
@artistgreen will review with @lkoenigsberg and @Samara-Strauss  gather feedback and finalize wireframes in sprint 8.

**Comment by Samara-Strauss** [comment]
Idk if this is ready for review, but if it is, can we please have a call to to discuss the design approach? Since this is a more complex flow, it would be easier to discuss that way instead of trying to understand the potential directions via this issue.

**Comment by artistgreen** [comment]
Sure!  I'm finalizing screens today and will be ready for review by tomorrow - will reach out to schedule a call.  Thanks.

**Comment by Samara-Strauss** [comment]
Arthur and I met over Slack today. I wanted to get my design feedback documented here so it doesn't get lost. Here is what we talked about:

- We have very few emails we send now (two confirmation emails— one is an email if health care apps fail to submit, and the second is a confirmation for direct deposit). I think sending an email to make people aware  that we have flagged their address is a sound suggestion. However, we don't have much in the way of a strategy around notifications right now, so we'd need to discuss further whether we actually want to add a non-confirmation email to the notifications we have. I think the answer is probably yes, tbh, but notifications should continue to be a larger team convo.
- On the subject of emails, the emails we have are all plain text/HTML, so that should be reflected in the design.
- If we assume the flow is email > VA.gov, then we should send people directly from the email to the sign in screen instead of the content page on changing addresses. This flow should make sense given the context we provide in the email.
- We should remove the homepage notification. The style reflected in the homepage mockup is only used for serious alerts that affect a large number of people (eg. a VA is closed because of hurricane). Truthfully, we don't have a way to show personalized alerts on the homepage right now. This is a known issue and something I hope we'll be able to tackle on the auth exp team at some point.

Also, wanted to include some notes of what we talked about re: how bad addresses might affect pre-fill/forms.

- We need to determine whether the flow outlined above and the APO/FPO/DPO profile work is dependent or independent of work that may need to happen on our benefit applications. I'm not sure of the answer to this. Arthur should determine this through his conversations with Andrea/Yana and whoever else.
- While we know it would be ideal if anyone could update the address we have on file for them in the profile from a benefit application, this is probably out of scope and a separate project *except* potentially when it comes to bad addresses. Again, I don't have the answers here. The exact dependencies and what is in/out of scope will be a continuing conversation based on the discovery happening around benefit applications and this work.
